src/instructlab/cli/profile/list.py

```diff
@@ -0,0 +1,24 @@
+# Standard
+import os
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.profile.list import format_output, list_profiles
+
+
+@click.command()
+@clickext.display_params
+def profile_list():
+    """List available profiles"""
+    profiles_dir = os.path.join(os.path.dirname(__file__), "../../profiles")
```

Comment:
Is there a default constant we can use here instead of a relative path?

Reply:
seems no, I use another way